index.js

```diff
@@ -16,16 +16,25 @@ app.use(route.get('/db', initDB));
 app.use(route.get('/register', register));
 
 function *index() {
-  console.log(this.req);
   this.body = {'message': 'Hello world'};
 }
 
 function *initDB() {
   yield this.pg.db.client.query_('CREATE EXTENSION postgis;');
-  yield this.pg.db.client.query_('CREATE TABLE "users" ("id" SERIAL, "uuid" UUID NOT NULL, PRIMARY KEY ("id"));');
-  yield this.pg.db.client.query_('CREATE TABLE "check_ins" ("id" serial, "users.id" serial, PRIMARY KEY ("id"), FOREIGN KEY ("users.id") REFERENCES "users"("id") ON DELETE CASCADE ON UPDATE CASCADE);');
-  yield this.pg.db.client.query_('SELECT AddGeometryColumn(\'check_ins\', \'location\', 4326, \'POINT\', 2);');
-  this.body = '1'
+  yield this.pg.db.client.query_('CREATE TABLE "users" ("id" SERIAL, "uuid" UUID NOT NULL, PRIMARY KEY ("id"), UNIQUE ("uuid"));');
+  yield this.pg.db.client.query_('CREATE TABLE "places" ("id" serial, "name" text, "guid" UUID NOT NULL, PRIMARY KEY ("id"), UNIQUE ("guid"));');
+  yield this.pg.db.client.query_('SELECT AddGeometryColumn(\'places\', \'coordinate\', 4326, \'POINT\', 2);');
+  yield this.pg.db.client.query_('CREATE TABLE "check_ins" ("id" serial, "users.id" serial, "places.id" serial, "created_at" TIMESTAMPTZ NOT NULL, PRIMARY KEY ("id"), FOREIGN KEY ("users.id") REFERENCES "users"("id") ON DELETE CASCADE ON UPDATE CASCADE, FOREIGN KEY ("places.id") REFERENCES "places"("id") ON DELETE CASCADE ON UPDATE CASCADE);');
+
+  yield this.pg.db.client.query_('INSERT INTO places ("name", "coordinate", "guid") VALUES (\'Apple HQ\', ST_GeomFromText(\'POINT(37.3317115 -122.0323722)\', 4326), \'' + uuid.v4() + '\');');
+  yield this.pg.db.client.query_('INSERT INTO places ("name", "coordinate", "guid") VALUES (\'Facebook HQ\', ST_GeomFromText(\'POINT(37.4833149 -122.1517118)\', 4326), \'' + uuid.v4() + '\');');
+  yield this.pg.db.client.query_('INSERT INTO places ("name", "coordinate", "guid") VALUES (\'Google HQ\', ST_GeomFromText(\'POINT(37.4219999 -122.0862462)\', 4326), \'' + uuid.v4() + '\');');
+  yield this.pg.db.client.query_('INSERT INTO places ("name", "coordinate", "guid") VALUES (\'Twitter HQ\', ST_GeomFromText(\'POINT(37.776692 -122.4189706)\', 4326), \'' + uuid.v4() + '\');');
+
+  yield this.pg.db.client.query_('INSERT INTO users ("uuid") VALUES (\'' + uuid.v4() + '\') RETURNING "id", "uuid";');
```

Comment:
Line is too long.<br>'yield' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).